.github/workflows/nodejs.yml

```diff
@@ -3,36 +3,24 @@ name: Node CI
 on:
   pull_request: {}
   push:
-    branches-ignore:
-      - trying.tmp
-      - staging.tmp
+    branches:
+      - master
```

Comment:
Limiting to master else a PR gets a CI run from both `push` and & `pull_request` triggers